controllers/openstack_ansibleee_controller.go

```diff
@@ -102,43 +102,44 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 		Log.Error(err, fmt.Sprintf("Unable to acquire helper for  OpenStackAnsibleEE %s", instance.Name))
 		return ctrl.Result{}, err
 	}
+	// Bump the ObservedGeneration as the new Reconciliation started
+	instance.Status.ObservedGeneration = instance.Generation
```

Comment:
@dprince @abays FYI we bump `ObservedGeneration` at the beginning of the Reconciliation loop (as per [1]).

[1] https://github.com/openstack-k8s-operators/glance-operator/pull/501